package/cpp/rnskia/dom/props/FontProp.h

```diff
@@ -31,7 +31,9 @@ class FontProp : public DerivedProp<SkFont> {
             "Expected SkFont object or null/undefined for the Font property.");
       }
     } else {
-      setDerivedValue(nullptr);
+      auto font = std::make_shared<SkFont>(SkFont());
+      font->setSize(14);
```

Comment:
is there a way to get the default system font size? but this is def ok

Reply:
this is based on the default font size being 14 in matchFont().